### PR TITLE
move mimetype from module_files to media_type on files

### PR DESCRIPTION
**Pull request by reedstrm**
compatability for a refactor in cnx-archive https://github.com/Connexions/cnx-archive/pull/403

### CODE REVIEW

File: Products/RhaptosModuleStorage/ModuleDBTool.py [hunk 1]
```diff
@@ -231,7 +231,7 @@ def _getFileID(self,fileob):
             if r.file == fdata:
                 return r.fileid
         # Fell through, must be new bytes
-        res = self.sqlInsertFile(file = Binary(fdata))
+        res = self.sqlInsertFile(file = Binary(fdata), media_type=fob.content_type)
```
Comment (karenc) [inline Products/RhaptosModuleStorage/ModuleDBTool.py]: Is this supposed to be `fileob.content_type`?

File: Products/RhaptosModuleStorage/sql/db_schema.sql [hunk 2]
```diff
@@ -242,6 +242,8 @@ CREATE UNIQUE INDEX modulekeywords_module_ident_keywordid_idx ON
 CREATE TABLE files (
     fileid serial PRIMARY KEY,
     md5 text,
+    sha1 text,
+    media_type text,
```
Comment (pumazi) [inline Products/RhaptosModuleStorage/sql/db_schema.sql]: media_type in the cnx-archive schema is the last declared value on the files table. I'm not sure this is problem a problem unless we are doing series inserts somewhere...

File: Products/RhaptosModuleStorage/ModuleDBTool.py (hunk 1 above)
Comment (pumazi) [inline Products/RhaptosModuleStorage/ModuleDBTool.py]: Yeah, I believe you are correct. The variable looks to be passed in as ``fileob``.

**Comment by reedstrm** [comment]
Can we change the db migration to leave it in a state that is backwards compatible with legacy so we can do this stepwise.

**Pull request merged**